packages/cli/bin/expf.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+SOURCE=${BASH_SOURCE[0]:-$0}
+while [ -L "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR=$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )
+  SOURCE=$(readlink "$SOURCE")
+  [[ $SOURCE != /* ]] && SOURCE=$DIR/$SOURCE # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+DIR=$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )
+
+node "${DIR}/expf.mjs" "$@"
```

Comment:
I think we may need to disable warnings here. Also, we may want to build autocomplete here.